examples/end-to-end-session-based/02-End-to-end-session-based-with-Yoochoose-PyT.ipynb

```diff
@@ -60,7 +60,7 @@
    "id": "d8642b00",
```

Comment:
<p>I would number that as <code>2</code> instead of <code>1.2</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/548/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>